src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -32,5 +32,7 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
-
+    public getFeedbackToUser(String){
```

Comment:
Generally okay, but I believe you may have missed out a few other classes, especially the test classes that have errors during compilation (because they use result.feedbackToUser directly).